encoding/fft/fft.go

```diff
@@ -36,6 +36,11 @@ import (
 	"math/bits"
 )
 
+// IsPowerOfTwo returns true if the provided integer v is a power of 2.
+func IsPowerOfTwo(v uint64) bool {
```

Comment:
Now you have a duplicate of the preexisting method. Probably should either use the preexisting method, or move the preexisting method to a better spot if it cannot be used due to import cycles

Reply:
lol fml this function was already there in fft, just private. I'll make it use ours.
Personally find it weird that this function is under `encoding` though...

Reply:
Just made it private since there's a circular import between fft and encoding and didn't feel like making a bigger refactor for this: [251b227](https://github.com/Layr-Labs/eigenda/pull/1841/commits/251b227e7ce190672ea7dbafc2f152cdd031701d)